prism_compile.c

```diff
@@ -4337,7 +4525,7 @@ pm_compile_node(rb_iseq_t *iseq, const pm_node_t *node, LINK_ANCHOR *const ret,
 
                           int index = pm_lookup_local_index(iseq, scope_node, name);
                           int kw_bits_idx = table_size - body->param.keyword->bits_start;
-                          int keyword_idx = (int)(i + scope_node->hidden_variable_count);
+                          int keyword_idx = (int)i;
 
                           ADD_INSN2(ret, &dummy_line_node, checkkeyword, INT2FIX(kw_bits_idx + VM_ENV_DATA_SIZE - 1), INT2FIX(keyword_idx - 1));
```

Comment:
```suggestion
                          ADD_INSN2(ret, &dummy_line_node, checkkeyword, INT2FIX(kw_bits_idx + VM_ENV_DATA_SIZE - 1), INT2FIX(i - 1));
```

Should we just use `i` here? Or rename `i` to `keyword_idx`?